Pair programming: Collaborative work on Contact Form Section

AbayomiOlaoye opened the pull request:
## Highlights
This **PR** features:
- High degree of collaboration among partners
- User-experience-friendly interface and choice of input elements
- Proper use of HTML form validation
- Form action and method of communication between the server and the client

AbayomiOlaoye committed:
Create mark-up text for contact section

AbayomiOlaoye committed:
Display form as flex for better abstraction and good user experience

UsirusenIkon committed:
Update Contact form class

UsirusenIkon committed:
Placeholder update

Commit:
Add flex to social medai

Commit:
Add social media class
to style.css

AbayomiOlaoye committed:
Make changes to contacts styling

AbayomiOlaoye committed:
Fix merge conflict from multiple collaborators

AbayomiOlaoye committed:
Finalize contact form section

AbayomiOlaoye committed:
Declare max-width size for recent-works

franclobo commented on index.html:
```diff
@@ -202,5 +202,57 @@ <h3 class="br-header">Multi-Post Stories</h3>
+    <div class="container">
+      <div class="contact-form">
+        <h2 class="recent-works robo-head center-align">Contact us</h2>
+        <p class="text">
+          If you have an application you are interested in developing, a feature 
+          that you need built or a project that needs coding. 
+          I'd love to help with it.
+        </p>
```
- [ ] Kindly resize the font to be similar to the Figma template. We are using 1.25 rem = 20 px and Figma template uses 16 px = 1 rem. 🧾

franclobo reviewed (changes requested):
Hi @MindPrepDev and @UsirusenIkon , 

Good job so far!
Well done you both, congratulations!!! 👏🏽 👏🏽 👏🏽 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] Please, 🙏🏽 add your partner to the authors' section on the README file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AbayomiOlaoye committed:
Make changes as requested by code reviewer
Update README to feauture other contributors and change font size of the last p element

muneebulrehman reviewed (approved):
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
